.github/workflows/push.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Continuous Integration'
+on:
+  push:
+    branches-ignore:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
I copied this from `.travis-yml`, do we still need this?

Reply:
This just cuts down time running extra builds when we publish a new release. Due to the way Semantic Release works, tags will always be running the exact same code as `master` at the time they are created, so running all the tests again is a bit of a waste.

Does GitHub Actions have a parallelism limitation? If not then we could remove this if we want. The whole reason we needed it was that we were often waiting on the fairly limited parallelism of TravisCI.

Reply:
on the free plan it's 20 concurrent builds per account

Reply:
I could go either way, leave it in, take it out... ⚖️

Reply:
There's also a `tags-ignore` option:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-ignoring-branches-and-tags

Reply:
```diff
diff --git a/.github/workflows/push.yml b/.github/workflows/push.yml
index 6a65adf5..84a7afd4 100644
--- a/.github/workflows/push.yml
+++ b/.github/workflows/push.yml
@@ -1,8 +1,7 @@
 name: 'Continuous Integration'
 on:
   push:
-    branches-ignore:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+    tags-ignore: ['*']
 jobs:
   unit-tests:
     runs-on: ${{ matrix.os }}
```

Reply:
agh ok, so we can just use `on: push` 😆 
by default this will only be [Triggered when code is pushed to any branch in a repository](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-using-a-single-event).

it seems that you have to explicitly opt-in to running workflows on tags, which is nice.